fixtures/c.vmx

```diff
@@ -0,0 +1,99 @@
+.encoding = "UTF-8"
+bios.bootorder = "hdd,cdrom"
+checkpoint.vmstate = ""
+cleanshutdown = "TRUE"
+config.version = "8"
+displayname = "generic-ubuntu1804-vmware"
+ehci.pcislotnumber = "-1"
+ehci.present = "FALSE"
+extendedconfigfile = "generic-ubuntu1804-vmware.vmxf"
+floppy0.present = "FALSE"
+guestos = "ubuntu-64"
+guestos.detailed.data = "bitness='64' distroName='Ubuntu' distroVersion='18.04' familyName='Linux' kernelVersion='4.15.0-76-generic' prettyName='Ubuntu 18.04.4 LTS'"
```

Comment:
It seems another split on the value would do it, no need for the regular expression unless I'm missing something here.

Reply:
here is an example: https://play.golang.org/p/tCDvzaz_Hjg

Reply:
Yeah, that should do it. Didn't think of SplitN. Nice thing about the regular expression is you don't need to check for comments, empty values or unquote separately. That is assuming there are never values without quotes.

Reply:
They are also harder to maintain, more inefficient and often come with unexpected side effects.

Reply:
Well it might not be as efficient, but less code also leaves less margin for error. I don't necessarily agree on the maintainability or side effect aspect, but I can change the PR if you want.

Reply:
> Well it might not be as efficient, but less code also leaves less margin for error

Agreed, it is another reason why I prefer using `strings.SplitN()`, it solves the problem with the least amount of code involved.